app/views/posts/_post.html.slim

```diff
@@ -0,0 +1,27 @@
+- @ticket_id = "ticket-#{post.id}"
+- @id = random_dom_id
+.ticket_box class=@ticket_id id=@id
+	= link_to post, {remote: true} do
+		.container-fluid
+			.row
+				.col-md-3
+					.profile_container
+						img class='profilepic_small' src="#{post.user.image}?sz=100"
+						.ticket_name #{post.user.first_name}
+				.col-md-9 class='ticket_description'
+					div.ticket_icon
+						img src=image_path('speechbubble.png')
+						.num_comments #{post.comments.count}
+					div.ticket_icon.follow
+						- post_followed = post.follows.exists?(user_id:current_user)
+
+						img src=image_path('followcheck.png') style="display:#{:none unless post_followed};"
```

Comment:
dont' do this here - instead, set `class: active` on the object, and set some CSS rules that show it `display: none` unless it has the `.active` class.
ex:
```
.theThing {
  display: none;
  &.active { display: block; }
}
```